spec/grape/jsonapi/parameters/filter_spec.rb

```diff
@@ -134,6 +134,7 @@
     end
 
     let(:model) { double(:model) }
+    let(:query) { double(:query) }
```

Comment:
I made the change to be more clear about what the result of the method is.  In Mongoid, for example, when you run this method it should return a `Mongoid::Criteria` object instead of the model.  The tests were passing before because we anticipated no change in the result (aka: the model would return itself instead of the results of the query).